internal/isoutil/isoutil.go

```diff
@@ -228,3 +232,64 @@ func (h *installerHandler) VolumeIdentifier() (string, error) {
 
 	return strings.TrimSpace(string(volumeId)), nil
 }
+
+func (h *installerHandler) GetFileLocation(filePath string) (int64, error) {
+	fs, err := h.isoFS()
+	if err != nil {
+		return 0, err
+	}
+
+	isoFile, err := fs.OpenFile(filePath, os.O_RDONLY)
+	if err != nil {
+		return 0, errors.Wrapf(err, "Failed to open file %s", filePath)
+	}
+
+	myFile := isoFile.(*iso9660.File)
+	defaultSectorSize := int64(2 * 1024)
```

Comment:
Is this something that we can read from the iso or get from go-diskfs rather than assuming?

Reply:
Tried to fetch it from go-diskfs but seems it's only a private const:
https://github.com/diskfs/go-diskfs/blob/ba492710e2d846eba20431917605d262200b7b81/filesystem/iso9660/iso9660.go#L17
Maybe indeed worth adding some means to externalize it later on.